config_template.cfg

```diff
@@ -1333,6 +1333,26 @@ UQ_URLX= 0.1
 % Perturbation magnitude (float [0,1], default= 1.0)
 UQ_DELTA_B= 1.0
 %
+% --------------------- HYBRID PARALLEL (MPI+OpenMP) OPTIONS ---------------------%
+%
+% An advanced performance parameter for FVM solvers, a large-ish value should be best
+% when relatively few threads per MPI rank are in use (~4). However, maximum parallelism
+% is obtained with EDGE_COLORING_GROUP_SIZE=1, consider using this value only if SU2
+% warns about low coloring efficiency during preprocessing (performance is usually worse).
+% Setting the option to 0 disables coloring and a different strategy is used instead,
+% that strategy is automatically used when the coloring efficiency is less than 0.875.
+% The optimum value/strategy is case-dependent.
+EDGE_COLORING_GROUP_SIZE= 512
+%
+% Independent "threads per MPI rank" setting for LU-SGS and ILU preconditioners.
+% For problems where time is spend mostly in the solution of linear systems (e.g. elasticity,
+% very high CFL central schemes), AND, if the memory bandwidth of the machine is saturated
+% (4 or more cores per memory channel) better performance (via a reduction in linear iterations)
+% may be possible by using a smaller value than that defined by the system or in the call to
+% SU2_CFD (via the -t/--threads option).
+% The default (0) means "same number of threads as for all else".
+LINEAR_SOLVER_PREC_THREADS= 0
+%
```

Comment:
New options introduced in #853 and #861 added to config_template.

Reply:
:+1: